src/client/pkg/require/require.go

```diff
@@ -511,6 +511,7 @@ func logMessage(tb testing.TB, msgAndArgs []interface{}) {
 	if len(msgAndArgs) == 1 {
 		tb.Logf(msgAndArgs[0].(string))
 	}
+
```

Comment:
Was there a change here at some point, or did you just really feel that there needed to be a space here lol?